src/lib/components/workflow/workflows-summary-row-with-filters.svelte

```diff
@@ -23,14 +24,63 @@
   export let workflow: WorkflowExecution;
   export let timeFormat: TimeFormat | string;
   export let checkboxDisabled: boolean;
+  export let columns: string[];
 
   $: href = routeForEventHistory({
     namespace,
     workflow: workflow.id,
     run: workflow.runId,
   });
 
-  let showFilterCopy = false;
+  $: data = [
```

Comment:
Definitely open for discussion on potentially better ways we can handle dynamically loading table cells and headers 💬